shuttering/prod/hmcts-access-service-gov-uk.yml

```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```

Comment:
This one also shutters with A records. Need shutter_all_a set to true and shutter_all_cname set to false

Reply:
done